src/services/nnid/routes/settings.ts

```diff
@@ -0,0 +1,42 @@
+import express from 'express';
+import { getServerByClientID, getPNIDByPID } from '@/database';
+import { HydratedServerDocument } from '@/types/mongoose/server';
+import { HydratedPNIDDocument } from '@/types/mongoose/pnid';
+import { decryptToken, unpackToken, getValueFromHeaders } from '@/util';
+import { Token } from '@/types/common/token';
+//import { GetUserDataResponse } from 'pretendo-grpc-ts/dist/account/get_user_data_rpc';
+//import { getUserAccountData } from '@/util';
+
+const router: express.Router = express.Router();
+
+/* GET discovery server. */
+router.get('/profile', async function (request: express.Request, response: express.Response): Promise<void> {
+	response.setHeader('x-nintendo-whitelist', '1|http,mii.olv.pretendo.cc,,2|1|https,mii.olv.pretendo.cc,,2|1|http,cdn.olv.pretendo.cc,,2|1|https,cdn.olv.pretendo.cc,,2|1|https,kaeru.b-cdn.net,,2|1|https,cdn.jsdelivr.net,,2|1|http,kt-imogen.b-cdn.net,,2|1|http,youtube.com,,2|http,portal.cdn.pretendo.cc,,2|https,portal.cdn.pretendo.cc,,2|http,pretendo-cdn.b-cdn.net,,2|https,pretendo-cdn.b-cdn.net,,2|https,youtube.com,,2|http,.youtube.com,,2|https,.youtube.com,,2|http,.ytimg.com,,2|https,.ytimg.com,,2|http,.googlevideo.com,,2|https,.googlevideo.com,,2|https,youtube.com,/embed/,6|https,youtube.com,/e/,6|https,youtube.com,/v/,6|https,www.youtube.com,/embed/,6|https,www.youtube.com,/e/,6|https,www.youtube.com,/v/,6|https,youtube.googleapis.com,/e/,6|https,youtube.googleapis.com,/v/,6|http,maps.googleapis.com,/maps/api/streetview,2|https,maps.googleapis.com,/maps/api/streetview,2|http,cbk0.google.com,/cbk,2|https,cbk0.google.com,/cbk,2|http,cbk1.google.com,/cbk,2|https,cbk1.google.com,/cbk,2|http,cbk2.google.com,/cbk,2|https,cbk2.google.com,/cbk,2|http,cbk3.google.com,/cbk,2|https,cbk3.google.com,/cbk,2|https,.cloudfront.net,,2|https,www.google-analytics.com,/,2|https,stats.g.doubleclick.net,,2|https,www.google.com,/ads/,2|https,ssl.google-analytics.com,,2|http,fonts.googleapis.com,,2|fonts.googleapis.com,,2|https,www.googletagmanager.com,,2|http,miiverse.cc,,2|https,miiverse.cc,,2');
```

Comment:
I thought you said the applet didn't respect this header? Also it's got lots of URLs that are not needed here